docs/api/transactions/read.md

```diff
@@ -49,6 +49,28 @@ Method: GET
 Endpoint: {openhim_url}:8080/transactions/:transactionId
 ```
 
+## Read a specific body for a specific transaction
+
+The response received when reading a transaction only contains an ID (`bodyId`) to reference the request, response or any other body in the transaction details. To fetch these they must be fetched individually.
+
+```curl
+Method: GET
+Endpoint: {openhim_url}:8080/transactions/:transactionId/bodies/:bodyId
+```
+
+The OpenHIM supports [HTTP range requests](https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests) on this endpoint so that the user may stream parts of the transaction bodies for larger transactions. Range requests are supported with the following restrictions:
```

Comment:
```suggestion
The OpenHIM partially supports [HTTP range requests](https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests) on this endpoint so that the user may stream parts of the transaction bodies for larger transactions. Range requests are supported with the following restrictions:
```